test/e2e/puppeteer.js

```diff
@@ -153,6 +153,7 @@ const exceptionList = [
 	'webgpu_tsl_vfx_linkedparticles',
 	'webgpu_tsl_vfx_tornado',
 	'webgpu_textures_anisotropy',
+	'webgpu_backdrop_water',
```

Comment:
`webgpu_backdrop_water` seems to have some timing issues. When updating related code, the example tends to fail although nothing changes at the visuals. Adding it to the exception list for now.